CMakeLists.txt

```diff
@@ -68,6 +68,9 @@ endif()
 include(sdk/cmake/compilerflags.cmake)
 
 add_definitions(-D__REACTOS__)
+add_definitions(-DREACTOS_SOURCE_DIR="\\"${REACTOS_SOURCE_DIR}\\"")
+add_definitions(-DREACTOS_BINARY_DIR="\\"${REACTOS_BINARY_DIR}\\"")
```

Comment:
The separating backslashes will also work on a Linux-configured build?

Reply:
Those are not separating backslashes. They are escapes for the double quote characters.

Reply:
Since the travis-CI builds are on linux and have succeeded, I would dare to say yes. The backslashes are escapes, not path separaters btw.

Reply:
Ohh yes, thanks!